tests/lib/nrf_modem_lib/lte_net_if/src/main.c

```diff
@@ -411,6 +411,13 @@ void test_pdn_act_without_cereg_should_not_activate_iface(void)
 	TEST_ASSERT_TRUE(net_if_is_dormant(net_if));
 }
 
+static int pdn_mtu_1000(uint8_t cid, struct in_addr *dns4_pri,
```

Comment:
Please place the stub at the top of the file
```suggestion
static int pdn_dynamic_params_get_stub_mtu_1000(uint8_t cid, struct in_addr *dns4_pri,
```